client/src/components/Card.tsx

```
@@ -0,0 +1,25 @@
+import PropTypes from 'prop-types';
+
+interface CardProps {
+  imageSrc: string;
+  title: string;
+  description: string;
+}
+
+const Card: React.FC<CardProps> = ({ imageSrc, title, description }) => {
+  return (
+    <div className="flex flex-col items-center justify-center border border-gray-300 p-6 basis-full md:h-80 rounded-md shadow-lg hover:shadow-xl hover:bg-pink-100 bg-gray-50">
+      <img src={imageSrc} alt="Card Image" className="w-24 h-24 mb-4" />
```

Biome (1.9.4)

[error] Avoid the words "image", "picture", or "photo" in img element alt text.

Screen readers announce img elements as "images", so it is not necessary to redeclare this in alternative text.

(lint/a11y/noRedundantAlt)